torch/nn/modules/dropout.py

```diff
@@ -58,6 +58,48 @@ def forward(self, input: Tensor) -> Tensor:
         return F.dropout(input, self.p, self.training, self.inplace)
 
 
+class Dropout1d(_DropoutNd):
+    r"""Randomly zero out entire channels (a channel is a 1D feature map,
```

Comment:
same, doc strings can probably be shared between 1d/2d/3d

Reply:
also flagged